docs/docs/animations/reanimated.md

```diff
@@ -13,7 +13,7 @@ When using the `useSharedValueEffect` hook, the canvas will redraw whenever the
 ## Definition
 
 ```tsx
-useSharedValueEffect(callback: () => void, values: Reanimated.SharedValue<any>[]);
+useSharedValueEffect(callback: () => void, ...values: Reanimated.SharedValue<any>[]);
```

Comment:
Even though the "real" signature is as follows:

`useSharedValueEffect = <T = number>(cb: () => void, value: SharedValueTypeWrapper<T>, ...values: SharedValueTypeWrapper<T>[])`

This is done to force at least one dependant value - so agree that it makes sense to keep the documentation declaration in simplified form.